tests/cases/seniors_app/case_seniors_app_home.yaml

```diff
@@ -1128,4 +1128,343 @@ TestSeniorIpadShouldNotResetIfCarePartnerUpdatesParticipantProfile:
       Role: desktopChrome
   # Go back to the iPad and fill in the Post Call Survey.
     - Type: case
-      Value: SeniorGreatPostSurveyCallHelper
\ No newline at end of file
+      Value: SeniorGreatPostSurveyCallHelper
+
+# CXTB-466: Verify "Emergency Services" button in portal
+TestSeniorVerifyEmergencyServicesButton:
+  Vars:
+    EMS_OPTION: 'true'
+  Environment: Settings
+  Precases:
+    - LoginNeverAloneHelper
+    - SeniorCleanPrompts
+    - CarePartnerCleanCallQueueAndHangedCalls
+  Aftercases:
+    - TerminateApp
+  Roles:
+    - Role: localiOS
+      App: NeverAlone
+    - Role: desktopChrome
+      Capabilities:
```

Comment:
Capabilities is empty. Can you check if there's something missing, if not please delete it.

Reply:
Done